src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-inline: 3px;
+    display: block;
+  }
+
+  &__head {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__text {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+  }
+
+  &__rating {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-color;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin-top: 16px;
+    width: 100%;
+    background-color: $button-color;
+    border-radius: 5px;
+    color: $button-white;
+    font-size: 14px;
+    font-weight: 700;
+    height: 40px;
+    text-transform: uppercase;
+    border: 1px solid $button-color;
+
+    &:hover {
+      background-color: $button-white;
+      color: $button-color;
+      transition: all 300ms ease-in-out;
+    }
+  }
+
+  &:hover {
+    transform: scale(120%);
+    transition: transform 300ms ease-in-out;
```

Comment:
The `transform: scale(120%);` property will cause the card to scale beyond its initial size on hover, potentially causing layout shifts. This could be a UX issue, as it might overlap other elements. Make sure that this effect doesn't break the layout or consider a more subtle effect if necessary.